requirements/test-requirements.txt

```diff
@@ -17,3 +17,5 @@ matplotlib
 shap>=0.40
 plotly
 kaleido
+# The latest flask version requires werkzeug>=2.2.0
+werkzeug>=2.2.0
```

Comment:
Note: We don't need to add the requirement restriction into `core-requirement`, because "flask" package already including the dependency restriction `werkzeug>=2.2.0`, but, when`pip install -r test-requirements.txt`, the installed `werkzeug>=2.2.0` package is replaced with a lower version `werkzeug` (see https://github.com/mlflow/mlflow/runs/7718421068?check_suite_focus=true#step:8:453 )